dev/Materials/Acrylic/AcrylicBrush.cpp

```diff
@@ -595,6 +595,15 @@ winrt::CompositionEffectBrush AcrylicBrush::CreateAcrylicBrushWorker(
         }
     }
 
+#ifndef BUILD_WINDOWS
+    winrt::CompositionObject compositionObject{ acrylicBrush.try_as<winrt::CompositionObject>() };
```

Comment:
I don't believe this line is required.  I think C++/WinRT is smart enough to be able to do acylicBrush.Properties() directly (which you could probably then inline directly on the InsertScaler line).